src/WarcraftLegacies.Source/Quests/Dalaran/QuestTheramore.cs

```diff
@@ -53,6 +49,19 @@ protected override void OnComplete(Faction completingFaction)
         completingFaction.Player.RescueGroup(_rescueUnits);
```

Comment:
We don't want to use `Rescue` methods here; they do a bunch of things that only really make sense when they're used on invulnerable, hidden, Neutral Passive units. Better to use the `SetOwner` method which just changes owner.

This does mean you won't have access to the handy `RescueGroup`, so you'll have to do the iteration manually. Luckily you already have a great example of iteration on line 53. Let me know if you need help.

Reply:
@SavanticIO can i let you take care of this one?

Reply:
Made a PR for this. RescueGroup() used on line 49 and 51 I think is correct use since they rescue a group of units created by PrepareUnitsForRescue()